jvm-packages/jenkins/Jenkinsfile-Release

```diff
@@ -23,7 +26,7 @@ UBUNTU_DOCKER_IMAGE = "${REGISTRY}/opsh2oai/h2o-3-xgboost-build-${UBUNTU_14}"
 XGB_MAJOR_VERSION = '0.7'
 XGB_VERSION = "${XGB_MAJOR_VERSION}.${currentBuild.number}"
 if (env.BRANCH_NAME != PUBLISHABLE_BRANCH_NAME) {
```

Comment:
Why a fixed version?

Reply:
That's a temporary thing. I want to do a GitHub release for 0.7.4